apps/src/calc/token.js

```diff
@@ -83,9 +83,11 @@ Token.prototype.setStringRepresentation_ = function () {
     return;
   }
 
+  // use toLocaleString so that we get commas per thousands
+
   // at this point we know we have a jsnumber
   if (this.val_.isInteger()) {
-    this.nonRepeated_ = this.val_.toFixnum().toString();
+    this.nonRepeated_ = this.val_.toFixnum().toLocaleString('en-US');
```

Comment:
toFixNum returns an actual number. Might still be worth a quick cross browser check.